test/functional/http/get/node.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+describe('GET /api/node/constants', function () {
+
+	var constantsResponse;
+
+	before(function (done) {
+		http.get('/api/node/constants', function (err, res) {
```

Comment:
I'd prefer to continue using promisified functions from apiHelpers. I've done this all along the rest offunctional tests.